### PR TITLE
Design/cards game

### LINKED ISSUE #82: As a user, I want a better designed home page, so that the platform is pleasant to use
- [x] New design for the home page
- [x] Implement the new design

### PR DESCRIPTION
**Pull request by Supergamer1337**
## Key Features
- Implemented new design for home page!
- Closes #82
- ... It looks nice?

## Known Issues
- The Legend is not at the bottom of the screen, because making sure that happens is a PIA, and doesn't really work the way one expects it too if font size changes, or screen width changes too much etc.
- Release date is not just year, because we haven't implemented that feature, making the filter a little wide.
- Gloomhaven is everywhere, because we don't have a feature for uploading game images yet.

### Other notes
- The cards do not yet lead to the detail page, as it doesn't exist in the PR. A separate PR will be used to merge both later on, or as we rebase we fix it.

### Screenshot of the page
image

### CODE REVIEW

**Review by Oscariremma — DISMISSED** [review]
LGTM!

We should probably look at consolidating the different forms components at a later date as we currently have "duplicates".

**Review by Oscariremma — DISMISSED** [review]

**Review by Oscariremma — APPROVED** [review]

File: frontend/src/components/GameCard/GameCard.tsx [hunk 1]
```diff
@@ -13,32 +13,46 @@ interface GameCardProps {
...
-			<p>Owner: {game.owner}</p>
-
-			<PlayStatus game={game} />
-
-			<GameRating game={game} />
-
-			<BorrowGame game={game} />
-			<RemoveGame game={game} />
-		</li>
+		<Link href={`/game/${game.id}`} legacyBehavior>
```
Comment (GAsplund) [inline frontend/src/components/GameCard/GameCard.tsx:16]: What's the reason to use `legacyBehavior` here?

**Review by GAsplund — CHANGES_REQUESTED** [review]
Looks good overall, and I really love how it looks, but it has minor issues. See comment.

Also, the "Add Game" button is slightly offset, or too tall compared to the search field. This hurts my brain:
image

File: frontend/src/components/GameCard/GameCard.tsx (hunk 1 above)
Comment (Supergamer1337) [reply frontend/src/components/GameCard/GameCard.tsx:16]: I didn't want it to be an `<a />` tag

Comment (GAsplund) [reply frontend/src/components/GameCard/GameCard.tsx:16]: Had to test, and I see the issue why. Could be fixed in CSS, but this is cleaner.

**Comment by Supergamer1337** [comment]
> Looks good overall, and I really love how it looks, but it has minor issues. See comment.
> 
> Also, the "Add Game" button is slightly offset, or too tall compared to the search field.

Actual pain, but when divs randomly are larger than what they should be, you do what you gotta do

**Review by GAsplund — APPROVED** [review]
My brain pain has been cured

**Review by Oscariremma — APPROVED** [review]

**Pull request merged**